package.json

```diff
@@ -41,8 +41,12 @@
   "license": "MIT",
   "private": true,
   "devDependencies": {
-    "jshint": "~2.5.2",
+    "fs": "0.0.1-security",
```

Comment:
It shouldn’t be necessary to manually install `fs`, since it’s included by default with Node. I believe this package only exists to prevent a bad actor from taking it over.

Reply:
Yeah, that was a mistake.  Sort of beside the point of the experiment, though. :smile: